ani-cli

```diff
@@ -266,6 +268,10 @@ play_episode() {
                 nohup "$player_function" $skip_flag --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 &
             else
                 "$player_function" $skip_flag --force-media-title="${allanime_title}Episode ${ep_no}" "$episode"
+                mpv_exitcode=$?
+                if [ "$exit_after_play" = 1 ]; then
+                    exit "$mpv_exitcode"
+                fi
```

Comment:
This should be wrapped into `[ "$exit_after_play" = 1 ] && exit "$mpv_exitcode"`.

Reply:
Fixed the problem!